great_expectations_cloud/agent/agent.py

```diff
@@ -85,8 +85,8 @@ def __init__(self: Self):
         print(f"GX Agent version: {agent_version}")
         print(f"Great Expectations version: {great_expectations_version}")
         print("Initializing the GX Agent.")
-        self._config = self._get_config()
         self._set_http_session_headers()
+        self._config = self._get_config()
```

Comment:
This is very minor and wasn't the cause of this bug, but ensure that the `/agent-session` call includes the custom `User-Agent` header.